index.html

```diff
@@ -354,7 +354,6 @@ <h2 class="abtme-title">I'm always interested in hearing about new projects, so
 				</ul>
 			</footer>
 		</section>
-
 	</main>
 
 </body>
```

Comment:
- [ ] I think it would be better if you enabled navigation to various sections of the page using the `desktop version menu`. You can achieve this by giving the sections `id's` and using them as links.
 
![image](https://user-images.githubusercontent.com/98453979/221897877-a406c8c4-89b0-446a-932a-e865bd94822b.png)